brave-okhttp/README.md

```diff
@@ -0,0 +1,17 @@
+# brave-okhttp #
+
+Contributed by [Tristan Burch](https://github.com/tburch).
```

Comment:
Please remove. while certainly appreciate the help, we dont typically do this (even if some crept in)